library/alloc/src/vec.rs

```diff
@@ -194,6 +194,22 @@ use crate::raw_vec::RawVec;
 /// can be slow. For this reason, it is recommended to use [`Vec::with_capacity`]
 /// whenever possible to specify how big the vector is expected to get.
 ///
+/// In summary, a vector containing 1, 2 with capacity 4 can be visualized as:
```

Comment:
```suggestion
/// A vector containing the elements `'a'` and `'b'` with capacity 4 can be visualized as:
```

I like using chars instead of integers in these examples, as it clearly distinguishes data from "meta data" fields, like len or capacity.